File: Chat-Example/src/main/java/client/protocols/Whisper.java
package client.protocols;

import client.ConnectionToChatServerHandler;
import com.google.gson.JsonObject;
import hoten.serving.message.JsonMessageHandler;

public class Whisper extends JsonMessageHandler<ConnectionToChatServerHandler> {

    @Override
    protected void handle(ConnectionToChatServerHandler connection, JsonObject data) {
        String from = data.get("from").getAsString();
        String msg = data.get("msg").getAsString();
        connection.getChat().whisper(from, msg);
    }
}

File: chat-example/src/main/csharp/ChatClient/ChatClient/Protocol/Whisper.cs
using Newtonsoft.Json.Linq;
using Serving;
using System;

namespace ChatClient.Protocol
{
    class Whisper : JsonMessageHandler<ConnectionToChatServerHandler>
    {
        protected override void Handle(ConnectionToChatServerHandler connection, JObject data)
        {
            var from = data["from"].Value<String>();
            var msg = data["msg"].Value<String>();
            connection.Chat.Whisper(from, msg);
        }
    }
}
